pallets/did/src/did_details.rs

```diff
@@ -245,10 +246,10 @@ pub struct DidDetails<T: Config> {
 	/// the old attestation keys that have been rotated, i.e., they cannot
 	/// be used to create new attestations but can still be used to verify
 	/// previously issued attestations.
-	public_keys: BTreeMap<KeyIdOf<T>, DidPublicKeyDetails<T>>,
-	/// \[OPTIONAL\] The URL pointing to the service endpoints the DID
+	pub(crate) public_keys: BTreeMap<KeyIdOf<T>, DidPublicKeyDetails<T>>,
+	/// \[OPTIONAL\] The service endpoint details the DID
 	/// subject publicly exposes.
-	pub endpoint_url: Option<Url>,
+	pub service_endpoints: Option<ServiceEndpoints>,
```

Comment:
Isn't that a single service endpoint? Or are it three service endpoints since there are 3 URLs? I expect a collection when ever i see plural, but there is no collection here! 😨